modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -5500,8 +5499,6 @@ type TargetEnvironmentGroupSelectResult {
 }
 
 type TargetGroup {
-  # IDs of observations that use the same constraints
-  observationIds: [ObservationId!]!
```

Comment:
This wasn't being used in the client and it's redundant with observations/matches/id so I removed it (same with other group types).